larray/core/array.py

```diff
@@ -1840,8 +1840,7 @@ def _translate_axis_key_chunk(self, axis_key, bool_passthrough=True):
             raise ValueError("%s is not a valid label for any axis"
                              % axis_key)
         elif len(valid_axes) > 1:
-            # FIXME: .id
-            valid_axes = ', '.join(str(a.id) for a in valid_axes)
+            valid_axes = ', '.join(str(a) for a in valid_axes)
```

Comment:
this is not correct for anonymous axes (and part of the reason I never did the change yet). Now that I think of it, it is not very hard either. Something like:
```python
', '.join(axis.name if axis.name is not None else str(self.axes.index(axis)) for axis in valid_axes)
```
should do the trick.